experimental/Schemes/src/ToricBlowups/constructors.jl

```diff
@@ -238,9 +245,28 @@ function blow_up(v::NormalToricVarietyType, n::Int; coordinate_name::Union{Strin
   gens_S = gens(cox_ring(v))
   center_unnormalized = ideal_sheaf(v, ideal([gens_S[i] for i in 1:number_of_rays(v) if cones(v)[n,i]]))
   blown_up_variety = normal_toric_variety(star_subdivision(v, n))
-  rays_of_variety = matrix(ZZ, rays(v))
-  exceptional_ray = vec(sum([rays_of_variety[i, :] for i in 1:number_of_rays(v) if cones(v)[n, i]]))
-  return ToricBlowupMorphism(v, blown_up_variety, coordinate_name, exceptional_ray, exceptional_ray, center_unnormalized)
+
+  # minimal supercone coordinates
+  coords = zeros(QQ, n_rays(v))
+  for i in 1:number_of_rays(v)
+    cones(v)[n, i] && (coords[i] = QQ(1))
+  end
+  exceptional_ray_scaled = standard_coordinates(polyhedral_fan(v), coords)
+  exceptional_ray, scaling_factor = primitive_generator_with_scaling_factor(
+    exceptional_ray_scaled
+  )
+  coords = scaling_factor * coords
+
+  phi = ToricBlowupMorphism(
+    v,
+    blown_up_variety,
+    coordinate_name,
+    exceptional_ray,
+    exceptional_ray,
+    center_unnormalized
+  )
+  set_attribute!(phi, :minimal_supercone_coordinates_of_exceptional_ray, coords)
+  return phi
 end
```

Comment:
Maybe I misunderstand - where is the bug fix?

To me, it currently looks like you added the computation of the minimal supercone coordinates and set them as an attribute of the `ToricBlowupMorphism`. To me, this looks like a new feature/extension (and is perfectly fine!). But, given the title of the PR - I cannot but wonder how this fixes a bug?